src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -229,11 +228,8 @@ class certificates extends viewModelBase {
         return accessManager.default.getAccessColor(accessLevel);
     }
 
-    getAccessInfoText(dbName: string, accessLevel: databaseAccessLevel) {
-        return `<div class="text-left padding padding-xs">
-                   <strong>Name</strong>: ${dbName === "All" ? "All databases" : generalUtils.escapeHtml(dbName) }<br/>
-                   <strong>Access</strong>: ${accessManager.default.getAccessLevelText(accessLevel) }
-                </div>`
+    getAccessInfoText(accessLevel: databaseAccessLevel) {
+        return accessManager.default.getAccessLevelText(accessLevel);
```

Comment:
Did you remove the tooltip content on purpose ?

Reply:
done